### PR TITLE
Shelly 4Pro and Shelly Sense are not supported

### PR DESCRIPTION
**Pull request by chemelli74**
## Proposed change

Shelly 4Pro and Shelly Sense are not supported by this integration due to firmware limitation.
They are still based on CoAP v1.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/shelly.markdown [hunk 1]
```diff
@@ -122,4 +122,5 @@ Not all devices support all input events. You can check on [Shelly API Reference
 ## Known issues and limitations
 
 - Only supports firmware 1.8 and later
+- Shelly4Pro is not supported (device based on old CoAP v1 protocol)
```
Comment (bieniu) [inline source/_integrations/shelly.markdown]: I think we're using `Shelly 4Pro` name, with a space, in `aioshelly`.

Comment (chemelli74) [reply source/_integrations/shelly.markdown]: Done

**Review by bieniu — DISMISSED** [review]
LGTM 👍

**Review by thecode — APPROVED** [review]
LGTM

**Review by frenck — APPROVED** [review]
Thanks, @chemelli74 👍

**Pull request merged**